src/resolvers/taxonomyResolvers.ts

```diff
@@ -0,0 +1,259 @@
+/**
+ * Copyright (c) 2024-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { IArticleV2 } from "@ndla/types-backend/article-api";
+import { ISubjectPageData } from "@ndla/types-backend/frontpage-api";
+import { Node, TaxonomyContext } from "@ndla/types-taxonomy";
+import {
+  fetchArticle,
+  fetchChildren,
+  fetchLK20CompetenceGoalSet,
+  fetchLearningpath,
+  fetchNode,
+  fetchNodeByContentUri,
+  fetchNodeResources,
+  queryNodes,
+} from "../api";
+import {
+  GQLLearningpath,
+  GQLNode,
+  GQLNodeChildrenArgs,
+  GQLNodeCoreResourcesArgs,
+  GQLNodeSupplementaryResourcesArgs,
+  GQLQueryArticleNodeArgs,
+  GQLQueryNodeArgs,
+  GQLQueryNodeCollectionArgs,
+  GQLQueryNodeResourceArgs,
+  GQLQueryNodesArgs,
+  GQLTaxonomyContext,
+  GQLTaxonomyCrumb,
+  GQLTaxonomyEntity,
+  GQLWithArticle,
+} from "../types/schema";
+import { nodeToTaxonomyEntity } from "../utils/apiHelpers";
+import { filterMissingArticles, getArticleIdFromUrn, getLearningpathIdFromUrn } from "../utils/articleHelpers";
+
+export const Query = {
+  async node(
+    _: any,
+    { id, rootId, contextId }: GQLQueryNodeArgs,
+    context: ContextWithLoaders,
+  ): Promise<GQLNode | undefined> {
+    if (rootId) {
+      const children = await fetchChildren({ id: rootId, recursive: true }, context);
```

Comment:
Burde recursive være et option her? Mistenker at det kan føre til at vi henter mer data enn vi trenger.

Reply:
rootId og id har forskjellige behov. rootId brukes i tax for å plukke korrekt context på noden som returneres, så begge må til her.

Reply:
I topicResolvers så spesifiseres det at kun topics skal hentes, men det er implisitt i tax. Kan legge det til for at det skal være likt.